### PR TITLE
DT-473 Create gnome calendar snap for Core22/Gnome42

### PR DESCRIPTION
**Pull request by sergio-costas**
# Pull Request Template

## Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please check only the options that are relevant.

- [ ] General Maintenance
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

**Test Configuration**:

* OS (please include version):
* Any other relevant environment information:

## Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have checked my code and corrected any misspellings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
 
-TODO: Add links to badges. Here's the gnome-calculator badge links for example:
-*[![gnome-calculator](https://snapcraft.io/gnome-calculator/badge.svg)](https://snapcraft.io/gnome-calculator)
-[![gnome-calculator](https://snapcraft.io/gnome-calculator/trending.svg?name=0)](https://snapcraft.io/gnome-calculator)*
+## Known bugs
 
-Below you will find some instructions to help you contribute to this snap. The general workflow will be to submit PRs from your fork onto the "stable" branch. Once the pull request has been *submitted*, there is a GitHub action that will automatically launch a build of the snap that you can use for testing. And then once the PR has been *merged* the GitHub action will launch a build on launchpad that will be automatically uploaded to the snap store, in the candidate channel, for more extensive testing.
+Snap still doesn't allow to access to Gnome Control Center through DBus, so the options to configure online
+accounts from Gnome Calendar doesn't work.
```
Comment (hellsworth) [inline README.md:11]: Is there a bug number for this somewhere we could link to?

**Review by hellsworth — CHANGES_REQUESTED** [review]
Link to bug where GOA doesn't work, but otherwise looks great! Thanks!

File: README.md (hunk 1 above)
Comment (sergio-costas) [reply README.md:11]: The reason, IIRC, is that the accounts are configured from Gnome Control Center, so if you can't open it, you can't configure them.

Comment (sergio-costas) [reply README.md:11]: Ok, I've been reviewing the interfaces, and there is https://snapcraft.io/docs/accounts-service-interface , which I think should fix this... I'll modify it after the online reviews.

**Pull request merged**